app/models/region.rb

```diff
@@ -54,6 +54,23 @@ def log_payload
     attrs.symbolize_keys
   end
 
+  def self.sources
+    includes(:source).map(&:source)
+  end
+
+  def syncable_patients
+    Patient
```

Comment:
This should support different patients for different region types:
FG -> registered patients
Facility -> registered patients
Block -> registered + assigned + appointed